packages/astro/src/integrations/index.ts

```diff
@@ -61,6 +61,25 @@ function getLogger(integration: AstroIntegration, logger: Logger) {
 	return integrationLogger;
 }
 
+const serverEventPrefix = 'astro-dev-toolbar';
+
+export function getToolbarServerCommunicationHelpers(server: ViteDevServer) {
+	return {
+		send: <T>(event: string, payload: T) => {
+			server.hot.send(event, payload);
+		},
+		on: <T>(event: string, cb: (data: T) => void) => {
+			server.hot.on(event, cb);
```

Comment:
To avoid collisions, it may be worth apply the same prefixing logic as below? so something like `${serverEventPrefix}:${appId}:${event}`. Same with send

Reply:
I went back and forth on this, it was like this initially and I ended up reverting it because you kinda can't do cross app communication with the prefixes, but I guess you still can